src/receivers/HTTPReceiver.ts

```diff
@@ -638,8 +638,8 @@ export default class HTTPReceiver implements Receiver {
 
           // Serve a basic HTML page including the "Add to Slack" button.
           // Regarding headers:
-          // - Content-Type is usually automatically detected by browsers
           // - Content-Length is not used because Transfer-Encoding='chunked' is automatically used.
+          res.setHeader('Content-Type', 'text/html');
```

Comment:
Having "charset" as part of content-type is the best practice to safely support non-English language use cases.
We already do it in bolt-python and bolt-java: https://github.com/slackapi/bolt-python/blob/v1.11.2/slack_bolt/oauth/oauth_flow.py#L176

```suggestion
          res.setHeader('Content-Type', 'text/html; charset=utf-8');
```

Reply:
Ooooo good call!